### DISCUSSION TITLE
Keyboard key input (windows) in vr mode?

### DISCUSSION THREAD

**Discussion by salexx** [StereoKit/StereoKit#931] [Q&A]
The console window does not intercept keyboard events (Windows).
Entering keyboard keys (Windows) in VR mode, how can I do it?

**Comment by maluoi** [comment]
The console window doesn't catch input, but there will be a small window with a SK icon on it while in an OpenXR session, and this is the one that catches input! It should be focused by default, but if you're clicking around on other windows, may lose focus and stop capturing keyboard input.

**Comment by salexx** [comment]
Hm. My window doesn't open. The SK process is running. I have a Pico-4 headset. Tried it through SteamVR and through VDXR (VirtualDesktop). Windows 10.



![SKproc](https://github.com/StereoKit/StereoKit/assets/3436243/cd05ced6-f654-4021-8e93-6c00503e049b)
![ConsoleLog](https://github.com/StereoKit/StereoKit/assets/3436243/5e4ebeac-fdec-49af-a074-736b1fc2d35c)

**Comment by salexx** [comment]
So far I've solved the problem like this:

https://github.com/rvknth043/Global-Low-Level-Key-Board-And-Mouse-Hook

Easy to use library. Works great in combination with StereoKit + Windows 10.

(I only used keyboard input. I haven't tested the mouse.)
